pkg/brokerapi/schemas.go

```diff
@@ -39,5 +43,5 @@ type ServiceBindingSchema struct {
 // InputParameters represents a schema for input parameters for creation or
 // update of an API resource.
 type InputParameters struct {
-	Parameters interface{} `json:"parameters,omitempty"`
+	Parameters *runtime.RawExtension `json:"parameters,omitempty"`
```

Comment:
@nilebox Thanks for the tip.

I've replaced this with `json.RawMessage`. Using `[]byte` didn't work - unmarshalling threw an error "json: cannot unmarshal object into Go value of type []uint8".